.sauce/testcafe.yml

```diff
@@ -1,18 +1,37 @@
 apiVersion: v1alpha
-metadata:
-  name: Testing TestCafe Support
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: Release $CI_COMMIT_SHORT_SHA
-files:
-  - ./tests/e2e/testcafe/example.test.js
-suites:
-  - name: "saucy test"
-    match: ".*.(spec|test).[jt]s$"
-image:
-  base: saucelabs/stt-testcafe-node
-  version: latest
+kind: testcafe
 sauce:
   region: us-west-1
+  concurrency: 1 # Controls how many suites are executed at the same time (sauce test env only).
+  metadata:
+    name: Testing Testcafe Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+suites:
+  - name: "saucy test"
+    browserName: "firefox" # Although Testcafe supports triggering one test in multiple browsers, it'd better here to split them into every suite to indicate each suite has its own test point.
+    src:
+      - "*test.js" # test files glob
```

Comment:
Should this be `src: "*test.js"` ? The `-` means that it's an array.

Reply:
Aye, it's deliberately an array, because src can be a `string` or `array` based on the official config: https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#src

So just like we did with cypress (where it can be both string an array), we go with the more flexible of the two and always have an array specified.

Reply:
Actually, I see now that it can be an array too.